dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -175,13 +177,41 @@ public boolean accept(File dir, String name) {
         });
 
         if (resultsFiles == null || resultsFiles.length == 0) {
-            Assert.fail("No TCK test JUnit result files were found in the results directory which suggests the TCK tests did not run - check build logs.\n"
-                        + "ResultsDir: " + surefileResultsDir.getAbsolutePath());
+            Assert.fail("No TCK test JUnit result files were found in the results directory which suggests the TCK tests did not run\n"
+                        + "Errors found in mvnOutput were:\n" +
+                        getErrorsFromMvnOutput());
         }
 
         return Arrays.asList(resultsFiles);
     }
 
+    private String getErrorsFromMvnOutput() {
+        File resultsDir = new File("results");
+        if (!resultsDir.exists())
+            return "";
+
+        File[] mvnOutputFiles = resultsDir.listFiles(new FilenameFilter() {
+            @Override
+            public boolean accept(File dir, String name) {
+                return name.startsWith("mvnOutput");
+            }
+        });
```

Comment:
It might make sense here, rather than looking for all mvnOutput files, to call `getMvnTestOutputFile()` to get the file for the current test run. This would avoid muddling the results if the TCK project repeats tests and both repeats fail.

If you want to stick with the current approach, you should definitely use `getResultsDir()` instead of `new File("results")` and `MVN_FILENAME_PREFIX` instead of `mvnOutput`.